stubs/model.py

```diff
@@ -1556,24 +1518,28 @@ def monolithic_solve(self):
 
         # march forward in time and update time-dependent parameters
         self.forward_time_step()
-        fancy_print(
+        logger.info(
             f"Beginning time-step {self.idx} [time={self.t}, dt={self.dt}]",
-            new_lines=[1, 0],
-            format_type="timestep",
+            extra=dict(
+                format_type="timestep",
+                new_lines=[1, 0],
+            ),
         )
         self.update_time_dependent_parameters()
         if self.config.solver["use_snes"]:
-            fancy_print("Solving using PETSc.SNES Solver", format_type="log")
+            logger.debug("Solving using PETSc.SNES Solver", dict(format_type="log"))
```

Comment:
Change to logger.info for consistency?

Reply:
Fixed in [f8eaa1a](https://github.com/RangamaniLabUCSD/stubs/pull/82/commits/f8eaa1aaec9d83148ef6dac5741a4913832d799b)